src/App/Controllers/Modules/GetContent.php

```diff
@@ -94,11 +94,24 @@ protected function getData($dir)
      */
     protected function getFolderContent($folder, $rec = false)
     {
-        return $this->storageDisk->listWith(
-            ['mimetype', 'visibility', 'timestamp', 'size'],
-            $folder,
-            $rec
-        );
+        return $this->storageDisk->listContents($folder)->map(function ($attributes) {
+            $path = $attributes->path();
+            $name = substr($path, strrpos($path, '/'));
+            $extension = substr($name, strrpos($name, '.') + 1);
+            return [
+                'path' => $path,
+                'dirname' => str_replace("/$name", '', $path),
+                'basename' => $name,
+                'extension' => $extension,
+                'filename' => substr($name, 0, strrpos($name, '.')),
+                'timestamp' => $attributes->lastModified(),
+                'size' => $attributes->isFile() ? $attributes->fileSize() : 0,
+                'type' => $attributes->isFile() ? 'file' : 'dir',
+                'mimetype' => $attributes->isFile() ? ($attributes->mimeType() ?: $this->getMimeFromExtension($extension)) : 'folder',
```

Comment:
for this we can use https://packagist.org/packages/league/mime-type-detection which is a dep of flysystem https://github.com/thephpleague/flysystem/blob/e5d33a0dd64f486da72e27b19091e46248f71a66/composer.json#L21